Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.6.0
+VERSION ?= 1.0.0
```

Comment:
This should be bumped to version 0.7.0. There are more changes needed to merge in from the certified 0.6.0 branch before this is version 1.0

Reply:
A release bump should be in a completely separate PR and not in this PR at all.